content/docs-v1/collection-catalogues.md

```diff
@@ -20,107 +68,201 @@ object_display_order:
 
-There are two different ways you can format your object entry pages: `side-by-side` in which the image appears next to the object information or `landscape` in which the image appears above the object information.
+### Object Link
 
-You can determine the format by either adding `presentation: side-by-side` or `presentation: landscape` to each entry's page YAML. Or you can set a default format for all your entry pages in the `content/_data/config.yaml` by choosing either `true` or `false` for the `sideBySide` attribute.
+If the `object_list` includes a `link` value, a “View in Collection” button is generated. The text of this button can be customized with the `objectLinkText` attribute in the project’s `content/_data/config.yaml` file.
 
 ```yaml
 entryPage:
-  sideBySide: true
+  objectLinkText: View in Collection
+  ```
+
+### Thumbnail 
+
+If you are using the object grid feature described below and the figure associated with a particular object *does not* include the setting `zoom: true` in the `figures.yaml`, then you can create a small thumbnail image for your object by using the `thumbnail` attribute and including a path to that image as the value. The image should be small, approximately 300 pixels wide. 
+
+If the object figure is not set to `zoom: true`, you will need to include both the figure `id` and the `thumbnail` path.
+
+```yaml
+object_list:
+  - id: 1
+    artist:
+    year:
+    dimensions:
+    medium:
+    location:
+    thumbnail: 
+    figure: 
+      - id: "fig-1-1"
+      - id: "fig-1-2"
+      - id: "fig-1-3"
 ```
 
-{{< q-figure id="modern-entry-side" >}}
+## Generate Object Grids
 
-Whatever the presentation, you can further organize the content on the page by using Quire’s `{% accordion %}` shortcode to wrap the individual parts of the object entry into collapsible sections. Read more in the [*Accordions*](/docs-v1/accordions/) chapter of this guide.
+In a collection catalogue, there is typically a visual table of contents that unifies all the catalogue entries/objects. 
 
-{{< q-figure id="modern-entry-side-accordion" >}}
+[Add object grid screenshot]
 
+To create this visual table of contents, which we refer to as an object grid there are three key steps. Add `layout: object-grid` to a `.md` file, add an `object-card` list to your `objects.yaml` file, and add an `object-filters` list to the `objects.yaml` file.
 
-### Object Images
+{{< q-class "box warning" >}}
+- As described in the [*Page Types & Structure*](/docs-v1/pages/#create-section-landing-pages) section of the documentation, an `index.md` file is used within a subfolder to create a landing page for all the contents found in that directory. An approximation of an object grid can be created by adding `layout: table-of-contents` and `presentation: grid` to the `index.md file`. However, we recommend reserving `layout: table-of-contents` for `essay` pages only.
```

Comment:
Talking about `index.md` files here felt confusing since, as you say at the beginning of Step 1, an `index.md` file doesn't need to be used.